### ISSUE TITLE
Live Test Companion App Does Not Auto-Refresh Changes

### ISSUE THREAD

**Issue by MrMazzone** [thunkable/thunkable-issues#727]
### Description/Background
For about a little over a month now, my students' non-PRO Thunkable accounts do not auto-refresh when they make changes to the app.  My PRO Thunkable account does, even on the same device that did for the students.

I want to preface this bug report with: Maybe you (Thunkable team) have changed this to a PRO only feature?  I hope not, as it is a real bummer to close the app and reopen each time to see changes. 

### Platform
Only tested on X Android Companion

### Steps to Reproduce
1. Log into Thunkable non-PRO account
2. Connect with Android companion
3. Make change to app, like change background color
4. Change does not show on device.

### Expected Behavior
Updates to app should refresh the companion on PRO and non-PRO accounts. 

### Actual Behavior
As described above, companion does not auto-refresh for non-PRO accounts.

**Comment by josmas** [comment]
Hey @MrMazzone we have not made any changes to the Live app depending on user accounts. Can this be something else such as students using a code to log in or similar?

**Comment by MrMazzone** [comment]
@josmas Very possible.  It seems to be most prevalent when students use code to log in.  But, my PRO account does work fine when I login with code.  Sometimes 1 student out of approx 15 will have auto-refresh.  Most have not had it for a little over a month.  

The only other thing I can think of, other than PRO vs non-PRO, is that these students create an account minutes before working on the project. Could that be something?  Newly created accounts are not fully setup and auto-refresh is not working?

**Comment by josmas** [comment]
I don't think so, new accounts should work the same as old ones. Are you all on the same network? or are you teaching remotely?
Gut feeling is that the main variable here is most likely the network. New, old, PRO and free accounts all use the same code when it comes the live app. Connections made with a code do use a different implementation, so that might be something to look into.

@jane-d-alt would you know if we've had any similar reports lately?

**Comment by MrMazzone** [comment]
@josmas All on the same network.  Student laptops and the Android tablets. And, myself and the tablets.  I am doing these in-person. We are hybrid in-person and remote at my school. 

Might be just connection with code issue, but what is weird is it works for my account no problem every time.

**Comment by josmas** [comment]
That's odd indeed. I can only think of issues with the network, really, but those are really tricky to debug and figure out.

Do you pre-install the companions into the tablets? all with the same account?
When you say users create the account right before using it, that is the thunkable account, right? but the tablet already has the app pre-loaded?

@weihuali0509 any idea what might be going on here?

**Comment by MrMazzone** [comment]
@josmas Yes. App is preinstalled on tablets.  I try to keep it up-to-date.  Have ensure it was up-to-date multiple times before the class uses it. 

Yes, the Thunkable account is created before. We create the Thunkable account, they create their first project, and then we connect to the tablet.

**Comment by DrTedGit** [comment]
I have found the lag between updating the project and the live test update has been pretty slow over the last few month. It is expecially problematic when a number of changes are made rapidly.  If I change 10+ blocks in under a minute, it can take ~ 5 minutes for all of the changes to propagate to Live Test.
